pkg/utils/utils.go

```diff
@@ -238,3 +240,13 @@ func (u *utils) IsBMInfra() (bool, error) {
 
 	return infra.Status.PlatformStatus.Type == configv1.BareMetalPlatformType, nil
 }
+
+func (u *utils) GetClusterVersion() (string, error) {
```

Comment:
This is only used in the test, not sure if you miss a use of this func in main or you can use this function that's already in the test files